### ISSUE TITLE
"Weak" relationships?

### ISSUE THREAD

**Issue by JohnMaguire** [tobscure/json-api#95]
Thank you again for this library!

I am reading through the JSON-API spec, and am looking at [this section here](http://jsonapi.org/format/#document-resource-object-relationships) which states that "A "relationship object" **MUST** contain at least one of the following," where the list is `data`, `links`, and `meta`.

I'm wondering if there is any way to create a "spare" relationship. i.e. I know the ID of the relationship, but the data is not currently available to me -- currently, if I just try to throw this at `Resource`, it will call `getAttributes`, and error out. I'm wondering if there is a way to only include links to the related object.

**Comment by tobscure** [comment]
Has this been solved with #94 / 24b0e85?

If you pass in a non-object/array as the Resource data (i.e. a string or an integer), then this will be used as the ID for the resource identifier object, and getAttributes will not be called. Or if you only wish to include `links` or `meta`, and no `data`, then just pass null as the Relationship data.

**Comment by JohnMaguire** [comment]
```
	public function device($data)
	{
		return new Relationship(new Resource(
			$data['device_id'],
			new DeviceSerializer
		));
	}
```

Works great, thanks!
